internal/app/api/v1/testsuites.go

```diff
@@ -855,6 +855,16 @@ func (s TestkubeAPI) ListTestSuiteTestsHandler() fiber.Handler {
 	}
 }
 
+func getTestSuiteNameFilter(c *fiber.Ctx) testresult.Filter {
```

Comment:
Could take the `string` instead of whole `*fiber.Ctx`

Reply:
yeah just followed previous one as it could get Query or Params from context.

Reply:
changed